packages/x-date-pickers/src/DesktopNextTimePicker/DesktopNextTimePicker.tsx

```diff
@@ -0,0 +1,280 @@
+   * @DateIOType
+   *
+   * @template TError, TValue
+   * @param {TError} reason The reason why the current value is not valid.
+   * @param {TValue} value The invalid value.
+   */
+  onError: PropTypes.func,
+  /**
+   * Callback fired when the popup requests to be opened.
+   * Use in controlled mode (see open).
+   */
+  onOpen: PropTypes.func,
+  /**
+   * Callback fired when the selected sections change.
+   * @param {FieldSelectedSections} newValue The new selected sections.
+   */
+  onSelectedSectionsChange: PropTypes.func,
+  /**
+   * Callback fired on view change.
+   * @template View
+   * @param {View} view The new view.
+   */
+  onViewChange: PropTypes.func,
+  /**
+   * Control the popup or dialog open state.
+   * @default false
+   */
+  open: PropTypes.bool,
+  /**
+   * First view to show.
+   */
+  openTo: PropTypes.oneOf(['hours', 'minutes', 'seconds']),
+  /**
+   * Force rendering in particular orientation.
+   */
+  orientation: PropTypes.oneOf(['landscape', 'portrait']),
+  readOnly: PropTypes.bool,
+  /**
+   * The currently selected sections.
+   * This prop accept four formats:
+   * 1. If a number is provided, the section at this index will be selected.
+   * 2. If an object with a `startIndex` and `endIndex` properties are provided, the sections between those two indexes will be selected.
+   * 3. If a string of type `MuiDateSectionName` is provided, the first section with that name will be selected.
+   * 4. If `null` is provided, no section will be selected
+   * If not provided, the selected sections will be handled internally.
+   */
+  selectedSections: PropTypes.oneOfType([
+    PropTypes.oneOf(['day', 'hour', 'meridiem', 'minute', 'month', 'second', 'year']),
+    PropTypes.number,
+    PropTypes.shape({
+      endIndex: PropTypes.number.isRequired,
+      startIndex: PropTypes.number.isRequired,
+    }),
+  ]),
+  /**
+   * Dynamically check if time is disabled or not.
+   * If returns `false` appropriate time point will ot be acceptable.
+   * @param {number} timeValue The value to check.
+   * @param {ClockPickerView} view The clock type of the timeValue.
+   * @returns {boolean} Returns `true` if the time should be disabled
+   */
+  shouldDisableTime: PropTypes.func,
+  /**
+   * If `true`, the toolbar will be visible.
+   * @default `true` for mobile, `false` for desktop
+   */
+  showToolbar: PropTypes.bool,
+  /**
+   * The system prop that allows defining system overrides as well as additional CSS styles.
+   */
+  sx: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.func, PropTypes.object, PropTypes.bool])),
+    PropTypes.func,
+    PropTypes.object,
+  ]),
+  /**
+   * The value of the picker.
+   */
+  value: PropTypes.any,
+  /**
+   * Array of views to show.
+   */
+  views: PropTypes.arrayOf(PropTypes.oneOf(['hours', 'minutes', 'seconds']).isRequired),
```

Comment:
(not related to this PR)
Same here, I'm not sure the default is correct. Not sure about what it should be when views do not open by default

Reply:
Changing the views changes the format in the field
And at some point we will allow to override the view renderers to people could have views on desktop

I agree that it is of little interest now, but I would keep it as is